HomeUI/src/views/apps/marketplace/SharedNodesView.vue

```diff
@@ -0,0 +1,1837 @@
+                </h3>
+                <a
+                  :href="`zel:?action=pay&coin=zelcash&address=${titanConfig.fundingAddress}&amount=${stakeAmount}&message=${signatureHash}&icon=https%3A%2F%2Fraw.githubusercontent.com%2Frunonflux%2Fflux%2Fmaster%2Fflux_banner.png`"
+                  class="col"
+                >
+                  <img
+                    class="zelidLogin"
+                    src="@/assets/images/zelID.svg"
+                    alt="Zel ID"
+                    height="100%"
+                    width="100%"
+                  >
+                </a>
+              </div>
+            </b-card>
+          </div>
+        </tab-content>
+      </form-wizard>
+    </b-modal>
+  </div>
+</template>
+
+<script>
+import {
+  BAvatar,
+  BButton,
+  BCard,
+  BCardBody,
+  // BCardHeader,
+  BCardText,
+  BCardTitle,
+  BCol,
+  BFormInput,
+  BFormSelect,
+  BFormSelectOption,
+  BFormSpinbutton,
+  BMedia,
+  BMediaBody,
+  BModal,
+  BRow,
+  BTabs,
+  BTab,
+  BTable,
+  VBModal,
+  VBToggle,
+  VBTooltip,
+} from 'bootstrap-vue';
+import {
+  FormWizard,
+  TabContent,
+} from 'vue-form-wizard';
+import VuePerfectScrollbar from 'vue-perfect-scrollbar';
+import Ripple from 'vue-ripple-directive';
+import { useToast } from 'vue-toastification/composition';
+import ToastificationContent from '@core/components/toastification/ToastificationContent.vue';
+
+import 'vue-form-wizard/dist/vue-form-wizard.min.css';
+
+import {
+  ref,
+  computed,
+} from '@vue/composition-api';
+
+import axios from 'axios';
+
+import tierColors from '@/libs/colors';
+import DashboardService from '@/services/DashboardService';
+
+const qs = require('qs');
+const store = require('store');
+const timeoptions = require('@/libs/dateFormat');
+
+export default {
+  components: {
+    BAvatar,
+    BButton,
+    BCard,
+    BCardBody,
+    // BCardHeader,
+    BCardText,
+    BCardTitle,
+    BCol,
+    BFormInput,
+    BFormSelect,
+    BFormSelectOption,
+    BFormSpinbutton,
+    BMedia,
+    BMediaBody,
+    BModal,
+    BRow,
+    BTabs,
+    BTab,
+    BTable,
+
+    FormWizard,
+    TabContent,
+
+    // eslint-disable-next-line vue/no-unused-components
+    ToastificationContent,
+
+    // 3rd Party
+    VuePerfectScrollbar,
+  },
+  directives: {
+    Ripple,
+    'b-modal': VBModal,
+    'b-toggle': VBToggle,
+    'b-tooltip': VBTooltip,
+  },
+  props: {
+    zelid: {
+      type: String,
+      required: false,
+      default: '',
+    },
+  },
+  setup(props, ctx) {
+    // Use toast
+    const toast = useToast();
+    const showToast = (variant, title, icon = 'InfoIcon') => {
+      toast({
+        component: ToastificationContent,
+        props: {
+          title,
+          icon,
+          variant,
+        },
+      });
+    };
+
+    const tier = ref('');
+    tier.value = props.tier;
+    const userZelid = ref('');
+    userZelid.value = props.zelid;
+
+    const apiURL = 'http://titantest.runonflux.io:54978';
```

Comment:
Here is the duplicated variable.

Reply:
Sorry, not sure how that made it's way in there. Keep the titantest apiURL.